server/utils/ssr.tsx

```diff
@@ -201,25 +201,86 @@ export const generateMetaComponents = (metaProps: MetaProps) => {
 			.filter((item) => {
 				return item.isAuthor;
 			});
+		const contributors = attributions
+			.sort((foo, bar) => {
+				if (foo.order < bar.order) {
+					return -1;
+				}
+				if (foo.order > bar.order) {
+					return 1;
+				}
+				if (foo.createdAt < bar.createdAt) {
+					return 1;
+				}
+				if (foo.createdAt > bar.createdAt) {
+					return -1;
+				}
+				return 0;
+			})
+			.filter((contributor) => {
+				return (
+					contributor.isAuthor &&
+					contributor.roles &&
+					(contributor.roles[0] === 'Writing – Review & Editing' ||
+						contributor.roles[0] === 'Editor' ||
+						contributor.roles[0] === 'Series Editor')
```

Comment:
Since these roles are significant, they could be extracted into an array above like:
```ts
// This is a terrible variable name... best to choose something else 
const contributorAttributionRoles = ['Writing – Review & Editing', 'Editor', 'Series Editor'];
const getPrimaryRole = (contributor: types.Contributor) => contributor.roles?.[0]
```

Then you can simplify your filter function like:
```ts
.filter((contributor) => {
  const primaryRole = getPrimaryRole(contributor);
  return isAuthor && primaryRole && contributorAttributionRoles.includes(primaryRole);
})
```